src/freeze-dry/inline-images.test.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-env jest */
+/* eslint import/namespace: "off" */
+
+import * as inlineImages from 'src/freeze-dry/inline-images'
+import * as common from 'src/freeze-dry/common'
+
+describe('getUrlsFromSrcset', () => {
+    test('should return URL for srcset', () => {
+        expect.assertions(3)
+        const srcset = 'https://example.com/background.jpg 1x, https://example.com/background.jpg 2x, https://example.com/background.jpg 3x'
+        const urls = inlineImages.getUrlsFromSrcset(srcset)
+        urls.map(url => expect(url).toBe('https://example.com/background.jpg'))
+    })
+})
+
+describe('inlineImages', () => {
+    test('should call inlineUrlsInAttributes', async () => {
+        common.inlineUrlsInAttributes = jest.fn()
+        const rootElement = window.document.implementation.createHTMLDocument()
+        const docUrl = 'about:blank'
+        await inlineImages.default({rootElement, docUrl})
+        expect(common.inlineUrlsInAttributes).toHaveBeenCalledTimes(3)
+    })
+})
```

Comment:
These tests are necessary to check whether the code runs the way we want it to run. I know it is a bit restrictive, but I wrote the tests on the grounds of unit testing.